account_journal_activate_tracking/models/account_journal.py

```diff
@@ -0,0 +1,7 @@
+from odoo import models, fields
+
+class AccountJournal(models.Model):
+    _inherit="account.journal"
+    
+    name = fields.Char(tracking=True)
+    bank_account_id = fields.Many2one('res.partner.bank',tracking=True)
```

Comment:
Is `comodel_name` really needed, does it work?
```suggestion
    bank_account_id = fields.Many2one(tracking=True)
```